manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "Hide Amazon Cart",
-  "version": "0.0.6",
+  "version": "0.0.9",
```

Comment:
We jumped from 6 to 9 solely because Firefox add-ons doesn't allow you to upload multiple versions with the same version number, and apparently I had already uploaded 7 and 8. So 9 it is.